transforms/angle-brackets/index.js

```diff
@@ -26,9 +28,40 @@ function getOptions() {
   return options;
 }
 
+function helperNames(helperPaths) {
+  return [
+    ...new Set(
+      helperPaths
+        .filter(name => name.includes('/helpers/') || name.includes('/helper'))
+        .filter(name => !name.includes('/-'))
```

Comment:
Not sure I follow, why do we want to exclude things with `/-` in the name?

Reply:
No.  I want to go back to my initial implementation.  I was "dog fooding" what we had in the readme, but the more I thought about it, my first implementation was simple and it worked.  Plus I will remove the section in the readme me that describes this code.